docs/topics/deployment.rst

```diff
@@ -0,0 +1,29 @@
+.. _topics-deployment:
+
+==========
+Deployment
+==========
+
+The recommended way to deploy Scrapy projects to a server is through `Scrapyd`_.
+
+.. _Scrapyd: https://github.com/scrapy/scrapyd
+
+Deploying to a Scrapyd Server
+=============================
+
+You can deploy to a Scrapyd server using the `Scrapyd client <https://github.com/scrapy/scrapyd-client>`_. You can add targets to your ``scrapy.cfg`` file which can be deployed to using the ``scrapyd-deploy`` command.
+
+The basic syntax is as follows:
+
+    scrapyd-deploy <target> -p <project>
+
+For more information please refer to the `Deploying your project`_ section.
```

Comment:
It's probably a good idea to make it clear that this link will take you to Scrapyd Server Docs (to avoid someone clicking it while browsing Scrapy Docs and getting confused, since the sites are similar).